data/legacy/src/main/java/app/tivi/data/mappers/TraktStatusToShowStatus.kt

```diff
@@ -16,16 +16,18 @@
 
 package app.tivi.data.mappers
 
+import app.moviebase.trakt.model.TraktShowStatus
 import app.tivi.data.models.ShowStatus
-import com.uwetrottmann.trakt5.enums.Status
 import me.tatarka.inject.annotations.Inject
 
 @Inject
-class TraktStatusToShowStatus() : Mapper<Status, ShowStatus> {
-    override suspend fun map(from: Status) = when (from) {
-        Status.ENDED -> ShowStatus.ENDED
-        Status.RETURNING -> ShowStatus.RETURNING
-        Status.CANCELED -> ShowStatus.CANCELED
-        Status.IN_PRODUCTION -> ShowStatus.IN_PRODUCTION
+class TraktStatusToShowStatus() : Mapper<TraktShowStatus, ShowStatus> {
+
+    override suspend fun map(from: TraktShowStatus) = when (from) {
```

Comment:
> Note: When getting full extended info, the status field can have a value of returning series (airing right now), continuing (airing right now), in production (airing soon), planned (in development), upcoming (in development), pilot, canceled, or ended.

https://trakt.docs.apiary.io/#reference/shows/summary

Reply:
Thanks for the pointer!